Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -1246,7 +1246,7 @@ public class PBXProjGenerator {
                 case .sdk:
                     dependencies[dependency.reference] = dependency
                 case .framework, .carthage, .package:
-                    if isTopLevel || dependency.embed == nil {
+                    if isTopLevel || dependency.embed != true {
```

Comment:
Why is this change needed? And why is it now different from similar checks below?

Reply:
I think they should all change to`embed != true` now.

Reply:
Done